server/auth.go

```diff
@@ -421,6 +425,7 @@ func loginJellyfin(user *User, db *gorm.DB) (AuthResponse, error) {
 			dbUser.ThirdPartyAuth = resp.AccessToken
 			dbUser.Username = resp.User.Name
 			dbUser.Type = JELLYFIN_USER
+			dbUser.Country = &Config.DEFAULT_COUNTRY
```

Comment:
The `loginJellyfin` and `loginPlex` methods will create a user if they don't exist in our database, so added the default country for them too.